libs/e2e-mock/server/src/lib/mock-descriptor/mock-descriptor.ts

```diff
@@ -87,6 +87,36 @@ class MockDescriptor {
       },
     }
   }
+  // Add method for mocking PRE_BACKUP responses
+  public mockPreBackupResponses(path: string, backupId: number) {
```

Comment:
it should be a part of test or helper in e2e app

Reply:
added helper - mock-prebackup.ts, removed prebackup mock from
 mock-descriptors, updated test to use the added helper instead of writing it in mockdescriptor or inside the test